spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -88,23 +90,35 @@
           ],
           "dates": [{"date": "1-1-2021", "type": "other", "description": "When the item was originally created."}],
           "formats": "application/pdf",
-          "locations": [{"place": "Boston, Massachusetts, United States"}, {"place": "East Peoria, Illinois, United States"}]
-        },
+          "locations": [{"place": "Boston, Massachusetts, United States"}, {"place": "East Peoria, Illinois, United States"}],
+          "funding": [{
```

Comment:
This is based on what was taken from the production csv and then converted into a hash stored in the text file above.